kpi-tracker/grafana/dashboard.json

```diff
@@ -1321,6 +1444,6 @@
   "timezone": "",
   "title": "KPI tracker",
   "uid": "8dm2sVA4k",
-  "version": 35,
+  "version": 10,
```

Comment:
This is an odd change. What is it for?

Reply:
This is just something that Grafana does automatically